test/sequential/test-http-pipeline-flood.js

```diff
@@ -72,20 +88,13 @@ function child() {
   req = new Array(10241).join(req);
 
   conn.on('connect', function() {
+    //kill child after flooding
+    setTimeout(function() { conn.destroy(); }, common.platformTimeout(1000));
```

Comment:
Hmm, actually, the "failing for both 0.10.20 and 0.10.21" issue (when it should fail only for 0.10.20) seems to be there before I even introduce `child.kill()` so it looks like I have to find where I introduced that regression.